optional/vtd/bin/vtd-launch

```diff
@@ -254,9 +254,9 @@ def start(
         shutil.copytree(vtd_setup, dst, symlinks=True)
 
     elif vtd_setup_cloe is not None:
-        vtd_setup_cloe = Path(os.getenv("VTD_SETUP_DIR")) / f"{vtd_setup_cloe}.tgz"
+        vtd_setup_cloe = Path(os.getenv("VTD_SETUP_DIR")) / f"{vtd_setup_cloe}.tar"
```

Comment:
Add a comment in the code why we do this. This will save the next person from reverting this in a year.